src/common/util/promise-timeout.ts

```diff
@@ -1,4 +1,4 @@
-export const promiseTimeout = (ms: number, promise: Promise<any>) => {
+export const promiseTimeout = (ms: number, promise: Promise<any> | any) => {
```

Comment:
If an argument to `Promise.race` is not a Promise it will resolve to that value immediately, so this type is OK.